### ISSUE TITLE
[bug] enable auto merge is failing

### ISSUE THREAD

**Issue by alita-moore** [ethereum/EIP-Bot#35]
https://github.com/ethereum/EIPs/pull/4518#issuecomment-985950971

**Comment by MicahZoltu** [comment]
Another instance of it happening: https://github.com/ethereum/EIPs/runs/4473811179?check_suite_focus=true

**Comment by MicahZoltu** [comment]
Closing and re-opening the PR appears to resolve the issue.

**Comment by alita-moore** [comment]
superceeded by https://github.com/ethereum/EIP-Bot/issues/38

**Comment by alita-moore** [comment]
I was mistaken

**Comment by alita-moore** [comment]
anyway I spent some time looking into this and it turns out the issue is the service that github offers to enable auto-merge is not reliable. There are a lot of fail cases. The above was from running into one of them. 

I investigated some options
- utilizing delayed enabling
- using different auto-merge utilities

I ultimately found that using https://kodiakhq.com/ along with https://github.com/actions-ecosystem/action-add-labels was the best option. The exact flow will be
1. bot passes
2. ecosystem adds a label on the PR 'automerge'
3. kodiak tries to merge but is prevent until required checks are passing

The primary benefit here is that adding a label does not have fail cases (that I know of) and regardless of the state it will not go away, so the auto-merge functionality will always be enabled and not subject to the funky rules normally given to auto merging.

Basically, the solution was to use other tools to approximate the same behavior as github because github's tools are not adequately compatible with our use-cases (at least that I could find).

**Comment by alita-moore** [comment]
I am awaiting approval to integrate `Kodiak` into ethereum/EIPs and then I will mark this complete, but the hard part is over (I hope)

**Comment by alita-moore** [comment]
fix was implemented by https://github.com/ethereum/EIPs/compare/ca4dbcecb266702ec410af4cc329f8dabdf1767a...f8ac1799a985ada9a2c0ce9b5af629bf6eccda7e

While this fix may seem simple, it took a significant amount of investigation, failure, and testing to find a feasible solution. 

I hope this one sticks!!
